velox/type/DecimalUtil.h

```diff
@@ -173,11 +174,16 @@ class DecimalUtil {
     }
     // Check overflow.
     if (!valueInPrecisionRange(rescaledValue, toPrecision) || isOverflow) {
-      VELOX_USER_FAIL(
```

Comment:
Waiting for [PR](https://github.com/facebookincubator/velox/pull/5307/files) merged.